### PR TITLE
#337 Set default timeout on ISBN search.

### PR DESCRIPTION
**Pull request by xihh87**
I've reduced the scope of changes on #338 [as requested](https://github.com/manubot/manubot/pull/338#issuecomment-1183912661):

> So in the interest of quickly merging, can we reduce this PR (or a new one) to just address the requests that are known to timeout?
>
> _Originally posted by @dhimmel in https://github.com/manubot/manubot/issues/338#issuecomment-1183912661_**

This change is only configurable on source code in the global variable defined on isbn search file.

### CODE REVIEW

File: manubot/cite/isbn.py [hunk 1]
```diff
@@ -16,6 +18,9 @@ class Handler_ISBN(Handler):
     def inspect(self, citekey):
         import isbnlib
 
+        isbnlib.config.setthreadstimeout(seconds=timeout_seconds)
+        isbnlib.config.seturlopentimeout(seconds=timeout_seconds)
```
Comment (dhimmel) [inline manubot/cite/isbn.py:22]: slight preference for us to define a helper like `import_isbnlib` such that we don't have to repeat this configuration 3 times.

**Review by dhimmel — COMMENTED** [review]
other than comment looks good.

File: manubot/cite/isbn.py (hunk 1 above)
Comment (xihh87) [reply manubot/cite/isbn.py:22]: Is a helper preferable to using only a global import and config for the library at the top of the module?

This is an ISBN getting module anyway.

Comment (dhimmel) [reply manubot/cite/isbn.py:22]: > Is a helper preferable to using only a global import and config for the library at the top of the module?

Usually not, although in this case let's assume we added the lazy importing of isbnlib for a reason and keep it that way.

Comment (dhimmel) [reply manubot/cite/isbn.py:22]: @xihh87 did you want to switch this to use a helper so we can merge?

**Pull request merged**